src/screens/Inventory/AddInventory.tsx

```diff
@@ -31,24 +33,12 @@ const styles = () =>
 
 const NEW_PRODUCT_LABEL = '+ New Inventory Item';
```

Comment:
This also needs to be translated (it shows up at the bottom of the "Select" list and they choose it if they want to create a new type of inventory item.